examples/SonDisk_Proc3/SerialConnector_4.pde

```diff
@@ -0,0 +1,173 @@
+// SerialConnector_3
+
+
+/*
+Standard stuff to add
+## variables 
+   SerialConnector sc;
+
+## setup
+   sc = new SerialConnector(this,port);
+   sc.listPorts();
+
+## draw
+sc.drawToScreen();
+
+
+## void keyPressed() {
+  sc.KeyInputforConnection(key);
+}
+
+
+*/
```

Comment:
Please remove commented out code. If it is an example on how to use an API or something similar, please designate it so.

Reply:
code removed